### PR TITLE
[Docs] Update migrate.md

### PR DESCRIPTION
**Pull request by nameYULI**
#### What this PR does / why we need it:
Update migrate.md

#### Does this PR introduce a user-facing change?

YES

### CODE REVIEW

**Comment by zhaosimon** [comment]
thanks for your submission!  We will look at it and get back to you soon.

File: docs/i18n/cn/docusaurus-plugin-content-docs/current/quick_start/create_stateful/advanced/migrate.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ sidebar_label:  "卷的迁移"
 
 # 卷的迁移
 
-`Migrate` 迁移功能是 HwameiStor 中重要的运维管理功能，当应用绑定的数据卷所在节点副本损坏时，卷副本可以通过迁移到其他节点，并在成功迁移到新节点后，将应用重新调度到新节点，并进行数据卷的绑定挂载。
+数据卷迁移 (Volume Migration) 是 HwameiStor 的重要运维管理功能。应用挂载的数据卷可以从一个有状况或通过告警提示即将出现状况的节点卸载迁移到一个正常的节点。数据卷迁移成功后，相关应用的 `Pod` 也被重新调度到新的节点并将数据新卷绑定挂载。
```
Comment (windsonsea) [inline docs/i18n/cn/docusaurus-plugin-content-docs/current/quick_start/create_stateful/advanced/migrate.md:8]:
```suggestion
数据卷迁移 (Volume Migration) 是 HwameiStor 的重要运维管理功能。
应用挂载的数据卷可以从一个有状况或通过告警提示即将出现状况的节点卸载迁移到一个正常的节点。
数据卷迁移成功后，相关应用的 `Pod` 也会被重新调度到新的节点并绑定挂载新的数据卷。
```
可以像 en 一样，适当折行，方便以后修改和 review

File: docs/docs/quick_start/create_stateful/advanced/migrate.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ sidebar_label:  "Migrate Volumes"
 
 # Migrate Volumes
 
-The `Migrate` function is an important operation and maintenance management function
-in HwameiStor. When the copy of the node where the data volume bound to the application
-is located is damaged, the copy of the volume can be migrated to other nodes, and after
-successfully migrated to the new node, the application can be rescheduled to the new
-node and bind mount the data volume.
+Volume Migration is an important operation and maintenance management function of HwameiStor.
+Application-mounted data volumes can be unmounted & migrated from a node with errors or an
```
Comment (windsonsea) [inline docs/docs/quick_start/create_stateful/advanced/migrate.md:9]:
```suggestion
Application-mounted data volumes can be unmounted and migrated from a node with errors or an
```

**Review by windsonsea — COMMENTED** [review]
Thanks. Pick nits there.

File: docs/docs/quick_start/create_stateful/advanced/migrate.md (hunk 2 above)
Comment (zhaosimon) [reply docs/docs/quick_start/create_stateful/advanced/migrate.md:9]: good suggestion!

**Pull request merged**